donut-assignment.html

```diff
@@ -0,0 +1,113 @@
+<script>
+  function Store(name, traffic, percentEnter, donutsPurchased) {
+    this.name = name;
+    this.traffic = traffic;
+    this.percentEnter = percentEnter;
+    this.donutsPurchased = donutsPurchased;
+    this.hours = 11;
+    this.hourlyDonuts = [];
+    this.getDonuts = function getDonuts(min, max) { 
+      //returns random number in range array
+      var ranTraffic = Math.floor(Math.random() * (max - min + 1)) + min;
+      return ranTraffic;
+      //find how many customers come in based on random number chosen and multiply the result by donuts purchased per person
+      var customers = Math.floor((ranTraffic * (this.percentEnter / 100)) * this.donutsPurchased);
+      return customers;
+    };
+  }
+
+  var downtown = new Store("Downtown", [80, 220], 10, 4);
+  var capitolHill = new Store("Capitol Hill", [5, 45], 45, 2);
+  var southLakeUnion = new Store("South Lake Union", [180, 250], 5, 6);
+  var wedgewood = new Store("Wedgewood", [20, 60], 20, 1.5);
+  var ballard = new Store("Ballard", [25, 175], 33, 1);
+  var hrs = ["7 and 8 a.m.", "8 and 9 a.m.", "9 and 10 a.m.", "10 and 11 a.m.", "11 a.m. and 12 p.m.", "12 and 1 p.m.", "1 and 2 p.m.", "2 and 3 p.m.", "3 and 4 p.m.", "4 and 5 p.m.", "5 and 6 p.m."];
+
+  //running the code 11 times, once for each hour of operation -- and pushing values into array
+  for(i = 1; i <= 11; i++) {
+    downtown.hourlyDonuts.push(downtown.getDonuts(downtown.traffic[0], downtown.traffic[1])); 
+    capitolHill.hourlyDonuts.push(capitolHill.getDonuts(capitolHill.traffic[0], capitolHill.traffic[1]));
+    southLakeUnion.hourlyDonuts.push(southLakeUnion.getDonuts(southLakeUnion.traffic[0], southLakeUnion.traffic[1]));
+    wedgewood.hourlyDonuts.push(wedgewood.getDonuts(wedgewood.traffic[0], wedgewood.traffic[1]));
+    ballard.hourlyDonuts.push(ballard.getDonuts(ballard.traffic[0], ballard.traffic[1]));
+  };
+
+  //print out hour by hour donut purchases
```

Comment:
This would be a good spot to use a for loop - then you could use i in place of hand-writing the array index each time